azure-pipelines.yml

```diff
@@ -3,19 +3,67 @@
 
 # Build STL targeting x86, x64, arm, arm64
 
-jobs:
-- template: azure-devops/run-build.yml
-  parameters:
-    targetPlatform: x86
+stages:
+  - stage: Code_Format_Validation
+    displayName: "Code Format Validation"
+    jobs:
+      - job: Run_Code_Format_Validation
+        timeoutInMinutes: 90
+        displayName: "Run Code Format Validation"
+        pool:
+          name: StlBuild-2020-03-28

```

Comment:
Should we add a variable for the pool name and pass it to `run-build.yml` so we don't have to change this in two places when we recreate the agent pool?

Reply:
Was just doing that 😄 .